configs/0/config.md
Initialize:1<br>
16:0x 1 0 0 0 e 0 7 0 0 0 1 0 d 0 0 0 3 0 3 0 0 0 5 0 a 0 0 0 7<br>
17:0x 3 c f 0 c f 9 d f 0 5 f f e d 0 6 f 4 c f 0 7 f 410 f 0 8<br>
18:0x 1 6 0 0 0 0 7 6 0 0 f 0 d 6 6 0 f 0 1 0 0 0 f 0 6 6 0 0 f<br>
19:0x 7 6 1 0 6 1 d 1 1 0 d 1 0 f d 0 c 1 6 b 1 0 b 1 2 5 1 0 a<br>
20:0x 110 0 0 e 0 7 4 0 0 d 0 d c 8 0 b 013 4 0 0 9 012 8 0 0 7<br>
21:0x13 f f 0 c f 4 9 f 0 3 f a 3 f 0 0 f10 3 f 012 f12 c f 0 f<br>
22:0x f e 4 0 0 415 0 4 0 9 4 5 8 0 0 5 4 b 0 4 0 1 4 0 2 4 013<br>
23:0x 310 9 015 9 911 9 010 9 f1214 0 b 915 5 9 0 6 9 714 9 0 1<br>
24:0x 1 0 0 0 6 0 7 0 0 0 9 0 d 0 0 0 3 013 0 0 015 012 0 0 0 f<br>
25:0x 110 0 0 9 0 7 e 0 0 8 0 d c17 0 1 01318 0 013 0 3 8 0 0 c<br>
26:0x b 4 4 0 e 411 6 4 0 b 417 8 e 0 3 4 310 4 015 416 c 4 0 d<br>
27:0x a f 9 012 910 6 9 0 6 91618 f 018 9 112 9 0 f 9 6 6 9 0 6<br>
28:0x 5 8 8 01a 8 b10 8 011 81118 8 0 7 81718 8 019 812 c 8 0 f<br>
29:0x151a 1 0 6 11b c 1 0 6 1 41b 3 018 1 a 5 1 0 d 1 21c 1 0 2<br>
30:0x 1 6 0 018 0 718 0 0 3 0 d c12 015 01318 0 0 9 01212 0 01b<br>
31:0x131b 9 0 d 919 1 9 019 9 0 61c 0 c 9 61a 9 01e 91910 9 011<br>
32:0x 1 0 0 01e 0 7 0 0 0 1 0 d 0 0 013 013 0 0 0 5 0 a 0 0 017<br>
33:0x 4 3 f 0 0 f a 0 f 0 9 f101e 0 01b f16 0 f 0 c f 018 f 01e<br>
34:0x 3 c20 0 c20 91e20 0 520 f e1e 0172015 c20 0 720 41020 019<br>
35:0x1a 1 f 013 f20 9 f 0 3 f 311 8 015 f 918 f 0 4 f1221 f 016<br>
36:0x 118 0 012 0 718 0 021 0 d1818 0 f 013 0 0 021 0 618 0 0 f<br>
[1, 196608, 16, 0, 3, 16]<br>
[-41219999, -41219998, -41220000, -41219996, -41228022, -41220000, -41219993, -41219992, -41220000, -41228022, -41219989, -41220000, -41219987, -41219986, -41228022, -41219984, -41219983, -41220000, -41219981, -41228022, -41220000, -41219978, -41219977, -41220000, -41228022, -41219974, -41220000, -41219972, -41219971]<br>
0xffffffff00000001/T:<br>


configs/a/config.md
`IP: 1/100100100100100`<br>
`C: [Fraction(3, 2), 18446744073709551617, 16, 0, 3, 16]`<br>
`!:/!!`<br>
`\/`<br>
`!!`<br>
`46`<br>
`000000000000000000000000000000`<br>
`000000000000000000000000000000`<br>
`000000000000000000000000000000`<br>
`000000000000000000000000000000`<br>
`100000000000000000000000000000`<br>
`010000000000000000000000000000`<br>
`111000000000000000000000000000`<br>
`000100000000000000000000000000`<br>
`000110000000000000000000000000`<br>
`100101000000000000000000000000`<br>
`010111100000000000000000000000`<br>
`111100010000000000000000000000`<br>
`100010011000000000000000000000`<br>
`110011010100000000000000000000`<br>
`101010111110000000000000000000`<br>
`111111100001000000000000000000`<br>
`000000010001100000000000000000`<br>
`000000011001010000000000000000`<br>
`000000010101111000000000000000`<br>
`000000011111000100000000000000`<br>
`100000010000100110000000000000`<br>
`010000011000110101000000000000`<br>
`111000010100101111100000000000`<br>
`000100011110111000010000000000`<br>
`000110010001100100011000000000`<br>
`100101011001010110010100000000`<br>
`010111110101111101011110000000`<br>
`111100001111000011110001000000`<br>
`1/100100100100100:3/2:`<br>
`3/200200200200200:3/2:`<br>
`9/400400400400400:3/2:`<br>
`27/800800800800800:3/2:`<br>
`81/1601601601601600:3/2:`<br>
`243/3203203203203200:3/2:`<br>
`729/6406406406406400:3/2:`<br>
`2187/12812812812812800:3/2:`<br>
`6561/25625625625625600:3/2:`<br>
`19683/51251251251251200:3/2:`<br>
`59049/102502502502502400:3/2:`<br>
`177147/205005005005004800:3/2:`<br>
`531441/410010010010009600:3/2:`<br>
`1594323/820020020020019200:3/2:`<br>
`4782969/1640040040040038400:3/2:`<br>
`14348907/3280080080080076800:3/2:`<br>
`43046721/6560160160160153600:3/2:`<br>
`129140163/13120320320320307200:3/2:`<br>
`387420489/26240640640640614400:3/2:`<br>
`1162261467/52481281281281228800:3/2:`<br>
`3486784401/104962562562562457600:3/2:`<br>
`10460353203/209925125125124915200:3/2:`<br>
`31381059609/419850250250249830400:3/2:`<br>
`94143178827/839700500500499660800:3/2:`<br>
`282429536481/1679401001000999321600:3/2:`<br>
`847288609443/3358802002001998643200:3/2:`<br>
`2541865828329/6717604004003997286400:3/2:`<br>
`7625597484987/13435208008007994572800:3/2:`<br>
`22876792454961/26870416016015989145600:3/2:`<br>
`68630377364883/53740832032031978291200:3/2:`<br>
`205891132094649/107481664064063956582400:3/2:`<br>
`617673396283947/214963328128127913164800:3/2:`<br>
`1853020188851841/429926656256255826329600:3/2:`<br>
`5559060566555523/859853312512511652659200:3/2:`<br>
`16677181699666569/1719706625025023305318400:3/2:`<br>
`50031545098999707/3439413250050046610636800:3/2:`<br>
`150094635296999121/6878826500100093221273600:3/2:`<br>
`450283905890997363/13757653000200186442547200:3/2:`<br>
`1350851717672992089/27515306000400372885094400:3/2:`<br>
`4052555153018976267/55030612000800745770188800:3/2:`<br>
`12157665459056928801/110061224001601491540377600:3/2:`<br>
`36472996377170786403/220122448003202983080755200:3/2:`<br>
`109418989131512359209/440244896006405966161510400:3/2:`<br>
`328256967394537077627/880489792012811932323020800:3/2:`<br>
`984770902183611232881/1760979584025623864646041600:3/2:`<br>
`2954312706550833698643/3521959168051247729292083200:3/2:`<br>
`8862938119652501095929/7043918336102495458584166400:3/2:`<br>
`26588814358957503287787/14087836672204990917168332800:3/2:`<br>
`79766443076872509863361/28175673344409981834336665600:3/2:`<br>
`239299329230617529590083/56351346688819963668673331200:3/2:`<br>
`717897987691852588770249/112702693377639927337346662400:3/2:`<br>
`2153693963075557766310747/225405386755279854674693324800:3/2:`<br>
`6461081889226673298932241/450810773510559709349386649600:3/2:`<br>
`19383245667680019896796723/901621547021119418698773299200:3/2:`<br>
`58149737003040059690390169/1803243094042238837397546598400:3/2:`<br>
`174449211009120179071170507/3606486188084477674795093196800:3/2:`<br>
`523347633027360537213511521/7212972376168955349590186393600:3/2:`<br>
`1570042899082081611640534563/14425944752337910699180372787200:3/2:`<br>
`4710128697246244834921603689/28851889504675821398360745574400:3/2:`<br>
`14130386091738734504764811067/57703779009351642796721491148800:3/2:`<br>
`42391158275216203514294433201/115407558018703285593442982297600:3/2:`<br>
`127173474825648610542883299603/230815116037406571186885964595200:3/2:`<br>
`381520424476945831628649898809/461630232074813142373771929190400:3/2:`<br>
`1144561273430837494885949696427/923260464149626284747543858380800:3/2:`<br>
`3433683820292512484657849089281/1846520928299252569495087716761600:3/2:`<br>
`10301051460877537453973547267843/3693041856598505138990175433523200:3/2:`<br>
`30903154382632612361920641803529/7386083713197010277980350867046400:3/2:`<br>
`92709463147897837085761925410587/14772167426394020555960701734092800:3/2:`<br>
`278128389443693511257285776231761/29544334852788041111921403468185600:3/2:`<br>
`834385168331080533771857328695283/59088669705576082223842806936371200:3/2:`<br>
`2503155504993241601315571986085849/118177339411152164447685613872742400:3/2:`<br>
`7509466514979724803946715958257547/236354678822304328895371227745484800:3/2:`<br>
`22528399544939174411840147874772641/472709357644608657790742455490969600:3/2:`<br>
`67585198634817523235520443624317923/945418715289217315581484910981939200:3/2:`<br>
`202755595904452569706561330872953769/1890837430578434631162969821963878400:3/2:`<br>
`608266787713357709119683992618861307/3781674861156869262325939643927756800:3/2:`<br>
`1824800363140073127359051977856583921/7563349722313738524651879287855513600:3/2:`<br>
`5474401089420219382077155933569751763/15126699444627477049303758575711027200:3/2:`<br>
`16423203268260658146231467800709255289/30253398889254954098607517151422054400:3/2:`<br>
`49269609804781974438694403402127765867/60506797778509908197215034302844108800:3/2:`<br>
`147808829414345923316083210206383297601/121013595557019816394430068605688217600:18446744073709551617:`<br>
`2726591648041031716020496776335699045695932041407581770817/121013595557019816394430068605688217600:16:`<br>
`2726591648041031716020496776335699045695932041407581770817/7563349722313738524651879287855513600:0:`<br>
`1001001001001000`<br>
`10010010010009901`<br>
`100100100100098910`<br>
`1001001001000989000`<br>
`10010010010009889900`<br>
`100100100100098898901`<br>
`00100100100098898891`<br>
`1001001000988988811`<br>
`10010010009889888010`<br>
`100100100098898880000`<br>
`00100100098898879990`<br>
`1001000988988799801`<br>
`10010009889887997910`<br>
`100100098898879979000`<br>
`1001000988988799789901`<br>
`10010009889887997898910`<br>
`100100098898879978989000`<br>
`1001000988988799789889901`<br>
`10010009889887997898898910`<br>
`100100098898879978988989000`<br>
`00100098898879978988988990`<br>
`00098898879978988988980`<br>
`8898879978988988970`<br>
`898879978988988960`<br>
`898879978988988950`<br>
`898879978988988940`<br>
`898879978988988930`<br>
`898879978988988920`<br>
`898879978988988910`<br>
`898879978988988900`<br>
`98879978988988890`<br>
`98879978988988880`<br>
`98879978988988870`<br>
`98879978988988860`<br>
`8879978988988850`<br>
`8879978988988840`<br>
`8879978988988830`<br>
`8879978988988820`<br>
`8879978988988810`<br>
`8879978988988800`<br>
`8879978988988790`<br>
`8879978988988780`<br>
`8879978988988770`<br>
`8879978988988760`<br>
`879978988988750`<br>
`79978988988740`<br>
`9978988988730`<br>
`978988988720`<br>
`978988988710`<br>
`978988988700`<br>
`978988988690`<br>
`978988988680`<br>
`978988988670`<br>
`978988988660`<br>
`78988988650`<br>
`78988988640`<br>
`78988988630`<br>
`78988988620`<br>
`8988988610`<br>
`8988988600`<br>
`8988988590`<br>
`8988988580`<br>
`8988988570`<br>
`8988988560`<br>
`8988988550`<br>
`8988988540`<br>
`8988988530`<br>
`8988988520`<br>
`988988510`<br>
`88988500`<br>
`8988490`<br>
`988480`<br>
`988470`<br>
`988460`<br>
`988450`<br>
`988440`<br>
`988430`<br>
`988420`<br>
`88410`<br>
`88400`<br>
`88390`<br>
`88380`<br>
`8370`<br>
`8360`<br>
`8350`<br>
`8340`<br>
`8330`<br>
`8320`<br>
`8310`<br>
`8300`<br>
`8290`<br>
`8280`<br>
`270`<br>
`!!`<br>
`!`<br>


configs/e/config.py
T = "0xffffffff00000001/"
I = "9/3/1"
P = "/0."
from fractions import Fraction
from time import sleep
import base64

commands = [Fraction(3, 2), 196608, 16, 0, 3, 16]

user_input = input("Initialize:")

# Function to check if the input is an integer
def is_integer(s):
    try:
        int(s)
        return True
    except ValueError:
        return False

if is_integer(user_input):
    commands[0] = int(user_input)
else:
    try:
        user_fraction = Fraction(user_input)
        commands[0] = user_fraction
    except ValueError:
        print("Invalid input. Please enter a valid integer or fraction.")

if user_input == '1':
    commands[0] = Fraction(3, 2)
    print("Input changed to 3/2")
elif is_integer(user_input):
    commands[0] = int(user_input)
else:
    try:
        user_fraction = Fraction(user_input)
        commands[0] = user_fraction
    except ValueError:
        print()

Tip = [user_input, 196608, 16, 0, 3, 16]
print(Tip)

def r30(a, b, c):
    if (a, b, c) in [(0, 1, 1), (1, 0, 0), (1, 0, 1), (0, 1, 0)]:
        return 1
    else:
        return 0

def calculate_denominator():
    r = [int(x) for x in format(int(input("T:")), '08b')]
    cx, step = 180, 25
    c, cn = [0] * cx, [0] * cx
    c[cx // 2:cx // 2 + 8] = r

    last_value = None  # To store the last value printed by r30

    for _ in range(step):
        print(''.join(['0' if e == 0 else '1' for e in c[90:123]]))
        for i in range(cx):
            x = c[i - 1] if i > 0 else 0
            z = c[i + 1] if i < cx - 1 else 0
            cn[i] = r30(x, c[i], z)
        c = cn[:]
        last_value = int(''.join(['1' if e == 1 else '0' for e in c[90:122]]))

    return last_value

# Get the last value printed by r30
last_denominator = calculate_denominator()

# Update the Fraction object 'ip' with the last denominator value
ip = Fraction(255, last_denominator + 1)

commands = [Fraction(4, 3), 196608, 16, 0, 3, 16]

def slash(I, P):
    while I:
        print(I)
        if I[0] == '/':
            I = I[1:]
        else:
            for c in P:
                if c == "/":
                    I = I[1:]
                elif c == "\\":
                    I = c + I
                else:
                    break
            if not I:
                break

input_val = None

while ip:
    eip = -ip if input_val else ip
    if eip < 0:
        input_val -= 1
    ci = eip % len(commands)
    if ci < 0:
        ci += len(commands)
    cmd = commands[int(ci)]
    if isinstance(cmd, int):
        print(f"{cmd}")
        sleep(0.02)
    ip *= Fraction(cmd)

    if cmd == 3:
        break

def m(n):
    if n > 100:
        return n - 10
    return m(m(n + 11))

initial_hex_value = "100100100100100"  # Initial hex value for conversion to binary
q, r, w = bin(int(initial_hex_value, 16))[2:], '101110101011011101001110101110101110100000', 3

while q[w:]:
    if r[0] == '0':
        q = q[1:]
    else:
        r = r[1:] + r[0]
        if q[0] == '1':
            q += r[0]
    r = r[1:] + r[0]
    q = str(m(int(q)))

    # Convert Base64 to hexadecimal representation
    hex_address = int(q.encode('utf-8').hex(), 36)
    hex_address = "0x{:x}".format(hex_address)
    
    print(hex_address)


